x-pack/plugins/cases/common/constants/owners.ts

```diff
@@ -17,6 +17,19 @@ export const GENERAL_CASES_OWNER = APP_ID;
 
 export const OWNERS = [GENERAL_CASES_OWNER, OBSERVABILITY_OWNER, SECURITY_SOLUTION_OWNER] as const;
 
+export const getOwnerUIName = (owner: Owner) => {
```

Comment:
The `OWNER_INFO` contains an attribute called `label`. Maybe we can use it. The names are a bit different but we can change it to match what you have here. They are being shown in the case flyout and in the cases modal.